bigbluebutton-tests/playwright/parameters/customparameters.js

```diff
@@ -158,6 +159,144 @@ class CustomParameters extends MultiUsers {
+    await this.modPage.waitAndClick(e.stopScreenSharing);
+    await this.modPage.hasElement(e.actions, 'should display the actions button');
+    await this.modPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the moderator');
+    await this.userPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the attendee');
+    await this.userPage.wasRemoved(e.whiteboard, 'should not display the whiteboard for the attendee');
+  }
+
+  async hidePresentationOnJoinShareExternalVideo() {
+    await this.modPage.hasElement(e.actions, 'should display the actions button');
+    await this.modPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the moderator');
+    await this.userPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the attendee');
+    await this.userPage.wasRemoved(e.whiteboard, 'should not display the whiteboard for the attendee');
+
+    await this.modPage.waitAndClick(e.actions);
+    await this.modPage.waitAndClick(e.shareExternalVideoBtn);
+    await this.modPage.type(e.videoModalInput, e.youtubeLink);
+    await this.modPage.waitAndClick(e.startShareVideoBtn);
+
+    const modFrame = await this.modPage.getYoutubeFrame();
+    await modFrame.hasElement('video', 'should display the video frame');
+
+    const userFrame = await this.modPage.getYoutubeFrame();
+    await userFrame.hasElement('video', 'should display the video frame');
+
+    await this.modPage.waitAndClick(e.actions);
+    await this.modPage.waitAndClick(e.shareExternalVideoBtn);
+
+    await this.modPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the moderator');
+    await this.userPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the attendee');
+    await this.userPage.wasRemoved(e.whiteboard, 'should not display the whiteboard for the attendee');
+  }
+
+  async hidePresentationOnJoinPinSharedNotes() {
+    await this.modPage.hasElement(e.actions, 'should display the actions button');
+    await this.modPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the moderator');
+    await this.userPage.hasElement(e.restorePresentation, 'should display the restore presentation button for the attendee');
+    await this.userPage.wasRemoved(e.whiteboard, 'should not display the whiteboard for the attendee');
+
+    await this.modPage.waitAndClick(e.sharedNotes);
+    await this.modPage.waitAndClick(e.notesOptions);
+    await this.modPage.waitAndClick(e.pinNotes);
+    await this.modPage.hasElement(e.unpinNotes, 'should display the unpin notes button');
+    await this.userPage.hasElement(e.minimizePresentation, 'should display the minimize presentation button for the attendee');
+    await this.modPage.waitAndClick(e.unpinNotes);
```

Comment:
before clicking to unpin, certify that the notification is closed. otherwise, it fails when trying to perform the click:

![image](https://github.com/user-attachments/assets/cc6ef318-90ce-4159-b790-253aff4bf3fd)